accepted/2022/ReflectionInvoke.md

```diff
@@ -0,0 +1,354 @@
+# Reflection Invoke for 8.0 (draft \ in progress)
+#### Steve Harter
+#### December 15, 2022
+
+# Background
+For additional context, see:
+- [Developers using reflection invoke should be able to use ref struct](https://github.com/dotnet/runtime/issues/45152)
+- [Originally library issue; needs refreshing\updating](https://github.com/dotnet/runtime/issues/10057)
+
+The invoke APIs and capabilities have essentially remained the same since inception of .NET Framework. Basically, [`MethodBase.Invoke`](https://learn.microsoft.com/dotnet/api/system.reflection.methodbase.invoke):
+```cs
+public object? Invoke(object? obj, object?[]? parameters);
+
+public object? Invoke(
+    object? obj,
+    BindingFlags invokeAttr,
+    Binder? binder,
+    object?[]? parameters,
+    CultureInfo? culture);
+```
+which is implemented by [MethodInfo](https://learn.microsoft.com/dotnet/api/system.reflection.methodinfo) and [ConstructorInfo](https://learn.microsoft.com/dotnet/api/system.reflection.constructorinfo). Fields do not have an `Invoke()` and instead have `GetValue()` and `SetValue()` methods since there is no invokable code around field access.
```

Comment:
Assume that properties are implemented by `MethodInfo` on the accessors?

Reply:
Yep. Just clarified that.